src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -861,33 +861,33 @@ trait GoldTransforms extends SparkSessionWrapper {
 
   protected val sparkJobViewColumnMapping: String =
     """
-      |organization_id, spark_context_id, job_id, job_group_id, execution_id, stage_ids, cluster_id, notebook_id, notebook_path,
+      |organization_id, workspace_name, spark_context_id, job_id, job_group_id, execution_id, stage_ids, cluster_id, notebook_id, notebook_path,
       |db_job_id, db_id_in_job, db_job_type, unixTimeMS, timestamp, date, job_runtime, job_result, event_log_start,
       |event_log_end, user_email
       |""".stripMargin
 
   protected val sparkStageViewColumnMapping: String =
     """
-      |organization_id, spark_context_id, stage_id, stage_attempt_id, cluster_id, unixTimeMS, timestamp, date, stage_runtime,
+      |organization_id, workspace_name, spark_context_id, stage_id, stage_attempt_id, cluster_id, unixTimeMS, timestamp, date, stage_runtime,
```

Comment:
maybe it would be easier to have a lookup table & have view over the existing data?

Reply:
these are the views that select over the existing data. I thought about putting it into a lookup but seemed unnecessary for a single value per workspace. This is just exposing it in the presentation layer (which are built views)